docs/source/getting_started.md

```diff
@@ -0,0 +1,170 @@
+# Getting started
```

Comment:
Één vraag die ik nog heb na het lezen van dit stuk: hoe kan je zien welke componenten je hebt toegevoegd aan je pipeline, en eventueel in welke volgorde deze worden afgetrapt? Heb even in de spaCy documentatie gekeken en ik denk dat het de .pipeline attribute is; misschien nog zinnig om dat toe te voegen?